packages/collector/test/test_util/ProcessControls.js

```diff
@@ -61,7 +61,9 @@ class ProcessControls {
           const esmApp = files.find(f => f.indexOf('.mjs') !== -1);
 
           if (esmApp) {
-            opts.execArgv = [`--experimental-loader=${path.join(__dirname, '..', '..', 'esm-loader.mjs')}`];
+            opts.execArgv = isLatestEsmSupportedVersion(process.versions.node)
+              ? ['--import', `${path.join(__dirname, '..', '..', 'esm-register.mjs')}`]
```

Comment:
Question: We use two arguments for `--import` and one for `--experimental-loader`. I know that there are two different notations. Can you double check please that both work? `--import {ARG` and `--import={ARG}`